pytasks.py

```diff
@@ -0,0 +1,50 @@
+"""
+Task2.
+
+pytasks.
+"""
+
+
+def generate_numbers(n=20):
+    """Return a dictionary with square if keys"""
+    import math
+    dic = {}
+    for x in range(n):
+        dic[x + 1] = int(math.pow(x + 1, 2))
```

Comment:
we can write just x*x or x**2 or pow(x,2)(without math) here if we would iter in range (1,n+1)